### PR TITLE
modify create defaults for pass4symmkey change

### PR DESCRIPTION
**Pull request by jrigassio-splunk**
Makes indexer and search head pass4SymmKey values match the randomly generated indexer and search head secrets in the event that neither are present. Resolves a test failure in develop as a result of a change to splunk-ansible.

### CODE REVIEW

File: splunk/common-files/createdefaults.py [hunk 1]
```diff
@@ -25,7 +25,9 @@
 splunk_hec_token = os.environ.get("SPLUNK_HEC_TOKEN", None)
 splunk_password = os.environ.get("SPLUNK_PASSWORD", None)
 splunk_idxc_secret = os.environ.get("SPLUNK_IDXC_SECRET", None)
+splunk_idxc_pass4SymmKey = os.environ.get("SPLUNK_SHC_PASS4SYMMKEY", None)
```
Comment (nwang92) [inline splunk/common-files/createdefaults.py]: Should be `SPLUNK_IDXC_PASS4SYMMKEY`

File: splunk/common-files/createdefaults.py [hunk 2]
```diff
@@ -43,8 +45,13 @@ def random_generator(size=24):
     os.environ["SPLUNK_PASSWORD"] = random_generator()
 if not splunk_idxc_secret:
     os.environ["SPLUNK_IDXC_SECRET"] = random_generator()
+if not splunk_idxc_pass4SymmKey:
+    os.environ["SPLUNK_IDXC_SECRET"] = os.environ["SPLUNK_IDXC_SECRET"]
```
Comment (nwang92) [inline splunk/common-files/createdefaults.py]: Thinking about this more, maybe this section should first check `splunk_idxc_pass4SymmKey`, that way when do deprecate the old variable in the splunk-ansible code, we don't need to modify this script again. Something like
```
if splunk_idxc_pass4SymmKey:
    os.environ["SPLUNK_IDXC_PASS4SYMMKEY"] = os.environ["SPLUNK_IDXC_SECRET"] = splunk_idxc_pass4SymmKey
elif splunk_idxc_secret:
    os.environ["SPLUNK_IDXC_PASS4SYMMKEY"] = os.environ["SPLUNK_IDXC_SECRET"] = splunk_idxc_secret
else:
   # generate random

File: splunk/common-files/createdefaults.py (hunk 1 above)
Comment (jrigassio-splunk) [reply splunk/common-files/createdefaults.py]: https://www.youtube.com/watch?v=atGjfmqb_TE

File: splunk/common-files/createdefaults.py (hunk 2 above)
Comment (jrigassio-splunk) [reply splunk/common-files/createdefaults.py]: Makes sense

**Review by bb03 — APPROVED** [review]

**Review by arctan5x — APPROVED** [review]

**Review by nwang92 — APPROVED** [review]

**Pull request merged**